submissions/rubyAdeniregun/Database/init3.sql

```diff
@@ -0,0 +1,75 @@
+CREATE DATABASE funa_travels;
+
+use funa_travels;
+
+CREATE TABLE registered_passengers (
+id INT NOT NULL AUTO_INCREMENT,
+full_name VARCHAR(100) NOT NULL,
+sex VARCHAR(10) NOT NULL,
+age INT,
+created_at datetime,
+primary key(id)
+);
+
+SELECT * FROM registered_passengers;
+
+CREATE TABLE passengers_records (
+id INT NOT NULL AUTO_INCREMENT,
+registered_passengers_id INT NOT NULL,
+passenger_class TINYINT NOT NULL,
+passenger_ticket_number VARCHAR(100) NOT NULL,
+trip_fare DECIMAL (10,2) NOT NULL,
+assigned_cabin VARCHAR(100),
+number_of_parents_children_aboard INT NOT NULL,
+number_of_siblings_spouses_aboard INT NOT NULL,
+point_of_embarkation VARCHAR(100) NOT NULL,
+created_at DATETIME,
+PRIMARY KEY (id),
+FOREIGN KEY (registered_passengers_id) REFERENCES registered_passengers (id)
+);
+
+SELECT * FROM passengers_records;
+
+
+CREATE TABLE accident_records (
+id INT NOT NULL AUTO_INCREMENT,
+registered_passengers_id INT NOT NULL,
+passenger_survived_yes_or_no VARCHAR(10) NOT NULL,
```

Comment:
This column will have a value of `1` or `0` so using `varchar is not best for it